recipes/perl-class-accessor/recipe.yaml

```diff
@@ -0,0 +1,48 @@
+schema_version: 1
+
+context:
+  name: perl-class-accessor
+  version: 0.51
+  sha256: bf12a3e5de5a2c6e8a447b364f4f5a050bf74624c56e315022ae7992ff2f411c
```

Comment:
```suggestion
```
Similarly, we don't need to use this in `context`, since its only used once further down.